python_modules/libraries/dagster-components/dagster_components_tests/code_locations/dependency_on_dbt_project_location/defs/depends_on_dbt/definitions.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+import dagster as dg
+from dagster_components.components.dbt_project.component import get_asset_key_for_model
+from dagster_components.core.component import ComponentLoadContext
+
+DBT_COMPONENT_PATH = Path(__file__).parent.parent / "jaffle_shop_dbt"
+
+ctx = ComponentLoadContext.current()
```

Comment:
We need to figure out the relationship between `DefinitionsLoadContext` and `ComponentLoadContext` stat